### PR TITLE
Gas overlap

### LINKED ISSUE #16: overlap of the path increases gas density on the ground

### PR DESCRIPTION
**Pull request by StefanThoene**
closes #16 and #15

### CODE REVIEW

File: mustards_game/gas_cloud.py [hunk 1]
```diff
@@ -111,10 +113,10 @@ def degrade_gas(self):
...
                         and y != 0
                     ):
-                        self.positions[x - 1, y].gas_level = self.positions[x, y].gas_level / 4
-                        self.positions[x + 1, y].gas_level = self.positions[x, y].gas_level / 4
-                        self.positions[x, y - 1].gas_level = self.positions[x, y].gas_level / 4
-                        self.positions[x, y + 1].gas_level = self.positions[x, y].gas_level / 4
+                        self.positions[x - 1, y].gas_level += self.positions[x, y].gas_level / 12
+                        self.positions[x + 1, y].gas_level += self.positions[x, y].gas_level / 12
+                        self.positions[x, y - 1].gas_level += self.positions[x, y].gas_level / 12
+                        self.positions[x, y + 1].gas_level += self.positions[x, y].gas_level / 12
```
Comment (beliaev-maksim) [inline mustards_game/gas_cloud.py:119]: let's set 12 as a variable

File: mustards_game/gas_cloud.py [hunk 2]
```diff
@@ -139,13 +141,14 @@ def get_cloud_volume(self):
         :return:
         """
         total_volume = 0
-        for position in self.positions:
-            total_volume += self.positions[position].level
+        for x in range(0, int(SCREEN_WIDTH / GAS_SIZE)):
```
Comment (beliaev-maksim) [inline mustards_game/gas_cloud.py:144]:
```suggestion
        for x in range(int(SCREEN_WIDTH / GAS_SIZE)):
```

**Review by beliaev-maksim — CHANGES_REQUESTED** [review]

**Comment by beliaev-maksim** [comment]
@StefanThoene 
need to resolve conflicts

**Comment by StefanThoene** [comment]
@beliaev-maksim  done

**Review by beliaev-maksim — APPROVED** [review]

**Pull request merged**